board.rb

```diff
@@ -0,0 +1,73 @@
+# class Board
+class Board
+
+  DIMENSION = 3
+
+  attr_accessor :size, :table
+
+  def initialize(size)
+    @size = size < 3 ? DIMENSION : size
+    @table = Array.new(@size)
+    @size.times { |pos| @table[pos] = Array.new(@size) }
+  end
+
+  def graphic_table
+    counter = 0
+    element = division = ""
+
+    for i in (0...@size)
+      for j in (0...@size)
+      
+        if @table[i][j].nil?
+          element += '-' + '|'
+        else
+          element += @table[i][j].to_s + '|'
+        end
+        counter += 1
+      end
+      element += "\n"
+      division = '-' * (@size * 2) + "\n" if (counter % @size).zero?
+      element += division
+    end
+
+    element
+  end
+
+  def fill_table(position, symbol)
+    arr_pos = position_coverter position
+    if empty_position? arr_pos
+      @table[arr_pos[0]][arr_pos[1]] = symbol
+      return true
+    else
+      return false
+    end
+  end
```

Comment:
Here's an idea:
`@table[arr_pos[0]][arr_pos[1]]= symbol if empty_position? arr_pos`
`cell == symbol`

Even better, what if you define a method to get the content of a cell and one for writing a cell?
`write_cell(arr_pos, symbol) if empty_position? arr_pos`
`read_cell(arr_pos) == symbol`